wdl-lint/tests/lints/blanks-between-elements/source.wdl

```diff
@@ -0,0 +1,87 @@
+#@ except: DescriptionMissing, ImportWhitespace, InputSorting, LineWidth, MissingMetas, MissingOutput, MissingRuntime, Whitespace
+
+version 1.1
+
+import "baz" # following whitespace will be caught by ImportWhitespace rule
+
+import "qux" # following whitespace duplication is caught be Whitespace rule
+
+
+# test comment
+workflow foo {
+
+    # This is OK.(but the prior line is not)
+    # So is this.
+    meta {}
+    parameter_meta {}
+    # what about this comment?
+    input {}
+
+    String p = "pip"
+
+    
+    String q = "bar" # The following whitespace is allowable between private declarations
+
+    String r = "world"
+    String s = "hello" # following whitespace duplication is caught be Whitespace rule
+
+
+    call bar { input:
+        s = s
+    } # following whitespace duplication is caught be Whitespace rule
+
+
+    call bar as baz { input:
+        s = s
+    }
+    call bar as qux { input: # Calls must be separated by whitespace.
```

Comment:
I don't think i agree with this. The `workflows` repo currently does not abide by this rule, and we frequently group call statements. What's your reasoning here?

Reply:
I actually agree with you. It turns out that the comment is wrong. I don't require `whitespace` between calls. I don't check it at all. I leave it to the `Whitespace` rule to detect excessive whitespace.